### DISCUSSION TITLE
cambio url app produzione

### DISCUSSION THREAD

**Discussion by prog14719** [fattureincloud/api#338] [Q&A] [answered]
Buongiorno,
avendo la necessità di migrare un nostro applicativo web che utilizza le vostre API, devo cambiare gli url dell'app pubblica che ho creato in Fatture in Cloud. 

Quali sono i cambiamenti che devo apportare nel mio codice sorgente che effettua le chiamate API? 

Gli utenti che hanno autorizzato già Fatture in Cloud tramite OAuth non dovranno ri-autorizzare l'applicazione?

I token generati fino ad ora continueranno a funzionare?

Grazie

![Cattura](https://github.com/fattureincloud/api/assets/139454490/0020731b-9967-4ad3-8d9e-801f9c7e4f8c)

**Comment by valmoz** [comment, marked as answer]
Buongiorno @prog14719,
qualora dovesse essere sostituito anche il Redirect URL, allora sarà necessario rimpiazzare il suo valore all'interno della pagina dell'applicazione e anche all'interno del codice del flusso OAuth (il valore del redirect URL deve essere passato in vari passaggi del flusso, può trovare maggiori informazioni [qui](https://developers.fattureincloud.it/docs/authentication/code-flow/vanilla-code)).

Per quanto riguarda i vecchi token, essi continueranno a funzionare come in precedenza.

Grazie
Mauro

**Reply by valmoz** [reply]
Buongiorno @prog14719 possiamo chiudere la discussione?

Grazie
Mauro
